custom_components/babybuddy/services.yaml

```diff
@@ -21,108 +21,53 @@ services_children_add:
       selector:
         text:
 
-services_delete:
```

Comment:
add back services_delete

Reply:
For this service I can add it as a service to delete the last entry (like last feeding entry or last diaper change entry). I will have to test it but i think it is possible. I don't recommend including the option to delete a child from HA.
What are your thoughts on this?

Reply:
I agree with you. Let's leave that off.

In order to keep backwards compatibilty, what are your thoughts on adding optional entity ID field to allow selecting specific entry to delete, but default to last?

Reply:
I am not sure how the user can find the ID of a previous entry? This is not available anywhere in the sensors state or attributes.

Reply:
Sorry, again operating under the false assumption this was only a minor version bump. Disregard.